roles/ispmail-firewall/tasks/main.yml

```
@@ -0,0 +1,15 @@
+---
+- name: Install or update iptables-related packages
```

```suggestion
- name: Install iptables-related packages
```
(`state: present` prevents any package update)